submissions/SerhiiYakovenko/memory-pair-game/script.js

```diff
@@ -0,0 +1,128 @@
+const cards = [
+  {
+    name: "alt",
+    img: "img/alt.png",
+  },
+  {
+    name: "hanako",
+    img: "img/hanako.png",
+  },
+  {
+    name: "kerry_eurodyne",
+    img: "img/kerry_eurodyne.png",
+  },
+  {
+    name: "saburo",
+    img: "img/saburo.png",
+  },
+  {
+    name: "silverhand",
+    img: "img/silverhand.png",
+  },
+  {
+    name: "smasher",
+    img: "img/smasher.png",
+  },
+  {
+    name: "v-female",
+    img: "img/v-female.png",
+  },
+  {
+    name: "v-male",
+    img: "img/v-male.png",
+  },
+];
+
+const div = document.createElement("div");
+div.id = "board";
+document.body.appendChild(div);
+
+let mixedCards = cards.concat(cards).sort(function () {
+  return 0.5 - Math.random();
+});
+
+let firstFlip = "";
+let secondFlip = "";
+let turn = 0;
+let previousCard = null;
+
+const board = document.getElementById("board");
+const boardGrid = document.createElement("section");
+boardGrid.setAttribute("class", "boardGrid");
+board.appendChild(boardGrid);
+
+mixedCards.forEach(function (item) {
+  const card = document.createElement("div");
+  const front = document.createElement("div");
+  const back = document.createElement("div");
+
+  card.classList.add("card");
+  card.dataset.name = item.name;
+  front.classList.add("front");
+  back.classList.add("back");
+  back.style.backgroundImage = `url('${item.img}')`;
+
+  boardGrid.appendChild(card);
+  card.appendChild(front);
+  card.appendChild(back);
+});
+
+let match = function match() {
+  let selected = document.querySelectorAll(".selected");
+  selected.forEach(function (card) {
+    card.classList.add("match");
+  });
+  if (document.getElementsByClassName("card match").length == 16) {
+    setTimeout(alert("Congrats! You won!"), 3000);
+  }
+};
+
+let reset = function reset() {
```

Comment:
1. `let` ?
2. why you give a name to variable and for a function ?)
`const reset  = function () {...} ` or `const reset = () => {... }` will be enough

Reply:
For some reason though that it may be changed, so decided to use let. 
Now I see that was wrong, thanks.
Also removed redundant function name.
Thanks!